pkgs/development/python-modules/oslo-db/default.nix

```
@@ -2,44 +2,48 @@
 , buildPythonPackage
 , fetchPypi
 , alembic
+, debtcollector
 , oslo-config
 , oslo-context
 , oslo-utils
 , oslotest
 , pbr
+, psycopg2
 , sqlalchemy
-, sqlalchemy-migrate
+, stevedore
 , stestr
 , testresources
 , testscenarios
 }:
 
 buildPythonPackage rec {
   pname = "oslo-db";
-  version = "12.3.1";
+  version = "14.0.0";
 
   src = fetchPypi {
     pname = "oslo.db";
     inherit version;
-    hash = "sha256-Gd7FAkDwj7q1rnZzVOtZF1oUdz8CcdeDpX/KxJ5KbaE=";
+    hash = "sha256-nAipzYOOv/rSHrMBL64AKg93v5Vpb6RNBbG2OiJ+n8E=";
   };
 
   nativeBuildInputs = [ pbr ];
 
   propagatedBuildInputs = [
     alembic
+    debtcollector
     oslo-config
     oslo-context
```

```suggestion
    oslo-i18n
```
https://opendev.org/openstack/oslo.db/src/tag/14.0.0/requirements.txt#L8